src/main/java/org/apache/commons/lang3/reflect/MethodUtils.java

```diff
@@ -782,7 +782,7 @@ private static int distance(final Class<?>[] classArray, final Class<?>[] toClas
         }
         for (int offset = 0; offset < classArray.length; offset++) {
             // Note InheritanceUtils.distance() uses different scoring system.
-            if (classArray[offset].equals(toClassArray[offset])) {
+            if (Objects.isNull(classArray[offset]) || classArray[offset].equals(toClassArray[offset])) {
```

Comment:
Objects#isNull is meant to be used with lambdas, use a `==` test here instead.

Reply:
I have update the commit with suggested changes. Please have a look. thanks !

Reply:
@garydgregory Please review and merge if changes are correct